src/index.js

```diff
@@ -4,17 +4,16 @@ import Raven from 'raven-js'
 import { initProviders } from 'actions/providers'
 import Decimal from 'decimal.js'
 import React from 'react'
+import { ConnectedRouter } from 'react-router-redux'
 
 import ReactDOM from 'react-dom'
 import { Provider } from 'react-redux'
-import { browserHistory } from 'react-router'
-import { syncHistoryWithStore } from 'react-router-redux'
 import { AppContainer } from 'react-hot-loader'
 import 'less/style.less'
-import AppRouter from 'router'
+import AppRouter from 'routes'
 import initGoogleAnalytics from 'utils/analytics/init'
 import BackdropProvider from 'containers/BackdropProvider'
-import store from 'store'
+import store, { history } from 'store'
```

Comment:
is this the same as:

``` 
import { createBrowserHistory } from 'history';

const history = createBrowserHistory();
```
? 

I do not exactly remember but I think it is better to go with this option. Can you have a look?

Reply:
@apanizo It's created inside store.js because we need to pass it into the middleware. I thought that we have to use the same history object everywhere, right? This is why I did it